PF/Assignment4/hanoi.java

```diff
@@ -0,0 +1,35 @@
+import java.util.Scanner;
+
+// program of tower of hanoi
+public class hanoi {
+
+	private static Scanner in;
```

Comment:
ACTION: Declare Scanner in Main function. As we are using it in main function only.